### DISCUSSION TITLE
Create New Content Post from code

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#7527] [Q&A]
I am working on a new module that will allow to import content from a word document and create a new content post. I am using my own custom content type, but need some guidance on how to add the record to the database.

**Comment by hishamco** [comment]
Have a look to ContentManager.Create

**Comment by deanmarcussen** [comment]
I'm going to refer you to this discussion https://github.com/OrchardCMS/OrchardCore/discussions/6927
which contains a lot of information about how to create content items programatically.
